fourkitchens/drupalcon-2017-webvr-demo
# FKVR-22: Add correct favicon and t itle

# [FKVR-22](https://fourkitchens.atlassian.net/browse/FKVR-22)
**This PR introduces the following changes:**
- Corrects the 4K logo asset so that it's a nice png with no white background. This logo, for prod builds, is compiled into a favicon automatically.
- Changes the title of this app to `VR Playground`.

## Steps to Test
Really, this shouldn't need testing. The favicon stuff works (see http://vr.fourkitchens.com, you can see that the favicon build step works, this PR just swaps out the image for a better one).
